### DISCUSSION TITLE
Steps Container

### DISCUSSION THREAD

**Discussion by 1nfinite9** [shuding/nextra#1305] [Ideas]
Are there any plans to provide documentation around how to show numbered steps like on this [page](https://nextra.site/docs/docs-theme/start)?

**Comment by dimaMachina** [comment]
Currently they are used only in nextra.site but we plan to extract them to the docs theme too

**Reply by 1nfinite9** [reply]
Cool! Looking forward to using them when available. And thanks for a great template site!!

**Reply by Valcoholics** [reply]
+1

**Reply by pawelkrystkiewicz** [reply]
+1

**Reply by yuxxeun** [reply]
+1

**Reply by dimaMachina** [reply]
It’s already exported

**Reply by dimaMachina** [reply]
https://github.com/shuding/nextra/releases/tag/nextra-theme-docs%402.2.15

**Comment by dimaMachina** [comment]
https://nextra.site/docs/guide/built-ins/steps
